x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -132,13 +133,17 @@ const createBenchmarks = (
       .filter(isNonNullable);
   });
 
-export const defineGetBenchmarksRoute = (router: IRouter, cspContext: CspAppContext): void =>
+export const defineGetBenchmarksRoute = (router: CspRouter, cspContext: CspAppContext): void =>
   router.get(
     {
       path: BENCHMARKS_ROUTE_PATH,
       validate: { query: benchmarksInputSchema },
     },
     async (context, request, response) => {
+      if (!context.fleet.authz.fleet.all) {
+        return response.forbidden();
+      }
```

Comment:
In the future, we'll have more fine-grained controls like `context.fleet.authz.fleet.readPackagePolicies` which will be useful here. In the meantime, users need "All" access to the Fleet app